1.20.1 Port

MianReplicate opened the pull request:
Regeneration but 1.20.1. I'm sowwy if i butched ur code during this port (not sorry)

- Same functionality as 1.19.2
- Tested everything I could think of testing, if anything is broken please let me know
- Zinc Ore seems to have an extreme spawn rate but I believe it was like this in 1.19.2 too
- A lot of changes were just refactors, gui changes, itemgroup edits, damagesources, and datagen.
- literally spent the entire day doing this, praise me

MianReplicate committed:
Initial port

MianReplicate committed:
Port two
we doing gud i wanna kms tho

MianReplicate committed:
Port number three!!

So close AAAAAAAAAA

MianReplicate committed:
everything ported i think?

LoxiGoose commented:
nvm i found some bugs with the custom heart icons, will fix them tmrw when I can

MianReplicate committed:
Fixed custom UI

Jeryn99 committed:
- Fixed issue where some skins on the web server did not have timestamp ids

Jeryn99 committed:
- Fixed issues with build.gradle where if the user did not have valid credentials, the build would fail
-

Jeryn99 committed:
Whoooops, you need to put the CD in your computer

MianReplicate committed:
uhhh i made the hand ui no longer a dirt block and also i fixed the trait command for you that was previously broken in 1.19.2 ur welcome craig

damagesrcs and cut off arm are still not working but they were problems in 1.19.2 too so theyre not my problem lol

MianReplicate committed:
fixed the aw

Jeryn99 committed:
- Goodbye Forge, Hello NeoForge
- Fixed texturing issue where Alex and Steve were purple due to ResourceLocation change caused by Mojangs addition of new skins

Jeryn99 committed:
- Improved some code

Jeryn99 committed:
- Potential make build pass

Jeryn99 committed:
Meta data related stuff

Test
